client/src/pages/trainee/dashboard/index.tsx

```diff
@@ -27,13 +28,13 @@ const DashboardPage: React.FC = () => {
           </div>
         </div>
         <div className="flex border-t-[0.5px] w-full justify-center items-start">
-          <div className="flex flex-col p-4 container w-[962px]">
+          <div className="flex flex-col p-4 container w-[1138px]">
```

Comment:
```suggestion
          <div className="flex flex-col p-4 container w-full">
```

is there a reason to use specific width size instead of `w-full`? please let me know :bow2:

Reply:
I think Elyric based it on the design in Figma:
![image](https://github.com/framgia/sph-lms/assets/116238730/116a06b4-ed47-4f8f-a44d-6859a087fd6e)